### ISSUE TITLE
Hex prefixes for luckperms in top

### ISSUE THREAD

**Issue by TymekSL** [CodedRed-Spigot/PlayTimes#51]
Hex prefixes from luckperms not working in the top show. Wrote this in dm - no fix

**Comment by CodedRedGIT** [comment]
This was fixed in the latest version of PlayTimes. Do you have luckperms added to papi?

**Comment by TymekSL** [comment]
> This was fixed in the latest version of PlayTimes. Do you have luckperms added to papi?

Yes! I wrote to you lot of messages in the spigot and you totally ignore me

**Comment by CodedRedGIT** [comment]
Are you still experiencing this issue? I tested and could not reproduce. PlaceholdersAPI (Tested with luckperms) worked perfectly fine in top playtime. Make sure you have PlayTimes and Luckperms registered in PlaceholdersAPI.

**Comment by TymekSL** [comment]
https://prnt.sc/CQprG_d_weOv

**Comment by TymekSL** [comment]
@CodedRedGIT
